ote/src/cljs/ote/ui/tabs.cljs

```diff
@@ -0,0 +1,28 @@
+(ns ote.ui.tabs
+  "Simple custom tab component"
+  (:require [stylefy.core :as stylefy]
+            [ote.style.tabs :as style-tabs]))
+
+(defn tabs
+
+  "Simble tab component that updates app-state using given function.
+  We need to let the parent give the update function, because there are endless variations what tab component
+  should do when tab is changed.
+  Assumes that first tab is selected by default."
+
+  [update-fn tabs selected-tab]
```

Comment:
I would change the tab interface to `[tabs {:update-fn ... :selected-tab :foo} ...tabs...]` to be easy to extend later without changing all call sites.